### PR TITLE
Update ga-track-fold to ES6

### PR DESCRIPTION
**Pull request by dannyvink**
Updated to the latest ga-track dependency and converted syntax to ES6.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* eslint-env browser */
+import GATrack from 'ga-track';
+import { ready } from 'domassist';
+import Cookies from 'js-cookie';
```
Comment (Antonio-Laguna) [inline index.js]: We should use https://github.com/firstandthird/cookie-monster to handle cookies

File: test/ga-track-fold.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import GATrackFold from '../index';
+import test from 'tape-rollup';
+
+//ga stub
+window._gaq = { // eslint-disable-line no-underscore-dangle
```
Comment (Antonio-Laguna) [inline test/ga-track-fold.test.js]: Probably disable for entire file with `eslint-disable` on top

**Review by Antonio-Laguna — CHANGES_REQUESTED** [review]
Great work here @dannyvink 

Few minor comments

**Comment by dannyvink** [comment]
Changes made! @Belelros

**Review by Antonio-Laguna — APPROVED** [review]

**Comment by Antonio-Laguna** [comment]
LGTM

**Pull request merged**